addons/payment_adyen/models/payment_transaction.py

```diff
@@ -87,21 +87,23 @@ def _send_payment_request(self):
         _logger.info("payment request response:\n%s", pprint.pformat(response_content))
         self._handle_feedback_data('adyen', response_content)
 
-    def _send_refund_request(self, refund_amount=None, create_refund_transaction=True):
+    def _send_refund_request(self, amount_to_refund=None, create_refund_transaction=True):
         """ Override of payment to send a refund request to Adyen.
 
         Note: self.ensure_one()
 
-        :param float refund_amount: The amount to refund
+        :param float amount_to_refund: The amount to refund
         :param bool create_refund_transaction: Whether a refund transaction should be created or not
-        :return: None
+        :return: The refund transaction if any, or `None`
+        :rtype: recordset of `payment.transaction`
```

Comment:
```suggestion
        :return: The refund transaction if any
        :rtype: recordset of `payment.transaction`
```
It's better to be consistent with return types. In `payment`, please change the implementation to:
```py
if blablabla:
    return refund_tx
else:
    return self.env['payment.transaction']
```